docs/static/docs/api/org/jspecify/annotations/NullMarked.html

```diff
@@ -93,7 +93,7 @@ <h1 title="Annotation Interface NullMarked" class="title">Annotation Interface N
  specified otherwise. Using this annotation avoids the need to write <a href="NonNull.html" title="annotation interface in org.jspecify.annotations"><code>@NonNull</code></a> many
  times throughout your code.
 
- <p>For a guided introduction to JSpecify nullness annotations, please see the <a href="http://jspecify.org/docs/user-guide">User Guide</a>.
+ <p>For a more comprehensive introduction to JSpecify, please see our <a href="http://jspecify.org">web site</a>.
```

Comment:
```suggestion
 <p>For a more comprehensive introduction to JSpecify, please see our <a href="http://jspecify.org">website</a>.
```

Reply:
do I haaave to

it rattles my teeth like "to setup" and "do it everyday", despite the fact that in this case it's not actually *wrong*.

Reply:
Ahh you gave me an out, below. Thank you.